packages/postgres-database/src/simcore_postgres_database/models/funcapi_function_job_collections_table.py

```diff
@@ -0,0 +1,51 @@
+"""Functions table
+
+- List of functions served by the simcore platform
+"""
+
+import uuid
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.sql import func
+
+from .base import metadata
+
+function_job_collections_table = sa.Table(
+    "funcapi_function_job_collections",
+    metadata,
+    sa.Column(
+        "uuid",
+        UUID(as_uuid=True),
+        default=uuid.uuid4,
+        primary_key=True,
+        index=True,
+        doc="Unique id of the function job collection",
+    ),
+    sa.Column(
+        "title",
+        sa.String,
+        doc="Title of the function job collection",
+    ),
+    sa.Column(
+        "description",
+        sa.String,
+        doc="Description of the function job collection",
+    ),
+    sa.Column(
```

Comment:
We have helpers `column_created_datetime` and `column_modified_datetime` 

https://github.com/itisfoundation/osparc-simcore/blob/96d76714dec459bb30fe9918c31a00f8bca4ac94/packages/postgres-database/src/simcore_postgres_database/models/_common.py#L19-L36

They produce the same. I would rather align the timestamp columns so they are the same in the entire repo.

Reply:
I will check it out. I basically copied it this from an existing table.

Reply:
Done.